2024-07-26  11:23:33 Fri werkzeug INFO [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5001
2024-07-26  11:23:33 Fri werkzeug INFO [33mPress CTRL+C to quit[0m
2024-07-26  11:23:41 Fri werkzeug INFO 127.0.0.1 - - [26/Jul/2024 11:23:41] "GET /duan HTTP/1.1" 200 -
2024-07-26  11:26:28 Fri basic ERROR Exception on /chao [GET]
Traceback (most recent call last):
  File "C:\Users\duanm\anaconda3\envs\development\lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\duanm\anaconda3\envs\development\lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\duanm\anaconda3\envs\development\lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\duanm\anaconda3\envs\development\lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "C:\Users\duanm\Music\GitHubProjects\PyDevelopment\flask_frame_note\basic.py", line 24, in hello_python
    a = 1 / 0
ZeroDivisionError: division by zero
2024-07-26  11:26:28 Fri werkzeug INFO 127.0.0.1 - - [26/Jul/2024 11:26:28] "[35m[1mGET /chao HTTP/1.1[0m" 500 -
